examples/misc/lib/effective_dart/design_bad.dart

```diff
@@ -0,0 +1,149 @@
+// ignore_for_file: type_annotate_public_apis, unused_element, unused_local_variable
+
+import 'dart:math' as math;
```

Comment:
Feel free to import dart:math without a prefix. I would :)

Reply:
That is my preference as well. I kept the name because that is how the examples had been written. I'll drop the prefix.

Reply:
I've dropped it here and for the other Effective Dart pages.